### DISCUSSION TITLE
post/kakaopay-dr-03/

### DISCUSSION THREAD

**Discussion by giscus** [kakaopay/techlog-comments#80] [Announcements]
# post/kakaopay-dr-03/

2024년, 2년 동안 해왔던 일을 회고하고 카카오페이의 "좋은 개발 문화"에 대해 고민합니다. 일상의 개발 문화 활동에 주목하고, Autonomous란 단어를 올해 개발 문화의 키워드로 제안합니다.

https://tech.kakaopay.com/post/kakaopay-dr-03/

**Comment by trendnote** [comment]
카카오페이 개발 조직을 이해하는데 도움이 되는 글이네요 ^^

**Reply by pay-dan-dy** [reply]
도움이 되셨다니 기쁘네요^^

**Comment by haeriiii** [comment]
다음 글도 기대되네요

**Reply by pay-dan-dy** [reply]
내년에는 "일상의 개발 문화 활동에 감사를 표현하도록 장려하는 시스템"을 운영한 결과를 공유드릴 수 있도록 하겠습니다^^

**Comment by benelog** [comment]
동현님 반갑습니다!

**Reply by pay-dan-dy** [reply]
아앗 상혁님 반갑습니다!^^
